font-types/src/raw.rs

```diff
@@ -57,7 +71,15 @@ impl<T: Scalar> BigEndian<T> {
     pub fn new(raw: T::Raw) -> BigEndian<T> {
         BigEndian(raw)
     }
-    /// Read a copy of this type from raw bytes.
+
+    /// Attempt to construct a new raw value from this slice.
+    ///
+    /// This will always fail if the slice is the correct length.
```

Comment:
nit: always is not necessary.

Also ... shouldn't it say is *not* the correct length? E.g. Fails if the slice is not the correct length ?

Reply:
oh yes, good catch.